src/components/error-label/error.jsx

```diff
@@ -1,15 +1,13 @@
 import React from "react";
-import { useTheme } from "@pokt-foundation/ui";
 import IconError from "../../icons/iconError";
 import ErrorLabelContainer from "./container";
 
 export default function ErrorLabel({ message, show, ...props }) {
-  const theme = useTheme();
   return (
     <>
       {show ? (
         <ErrorLabelContainer {...props}>
-          <IconError color={theme.negative} className="icon" />
+          <IconError color="#F93232" className="icon" />
```

Comment:
Oof, is that color not included in the theme?

Reply:
Apparently not.